pkgs/top-level/python-packages.nix

```
@@ -2817,6 +2817,8 @@ in {
 
   ffmpeg-progress-yield = callPackage ../development/python-modules/ffmpeg-progress-yield { };
 
+  fiblary3 = callPackage ../development/python-modules/fiblary3 { };
```

```suggestion
  fiblary3-fork = callPackage ../development/python-modules/fiblary3-fork { };
```
Let's add `fiblary3` as an alias.